src/transformers/integrations/peft.py

```diff
@@ -30,6 +31,10 @@
 # Minimum PEFT version supported for the integration
 MIN_PEFT_VERSION = "0.5.0"
 
+if is_torch_available():
+    import torch
+
```

Comment:
```suggestion
if TYPE_CHECKING:
    if is_torch_available():
        import torch  # noqa
```
as it it' s not directly used

Reply:
done in [55d657b](https://github.com/huggingface/transformers/pull/25721/commits/55d657b072beddf4080cb2b44c0b33fafb8f7e87)